Apps/AdminWebClient/src/Server/Controllers/CovidSupportController.cs

```diff
@@ -0,0 +1,90 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Admin.Controllers
+{
+    using HealthGateway.Admin.Models.Support;
+    using HealthGateway.Admin.Services;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+
+    /// <summary>
+    /// Web API to handle Covid support requests.
+    /// </summary>
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Route("v{version:apiVersion}/api/[controller]")]
+    [Produces("application/json")]
+
+    // [Authorize(Roles = "AdminUser")]
+    public class CovidSupportController
```

Comment:
This one needs to get the correct role, I think once we hook up the front end as well this can be changed to make sure everything works

Reply:
SupportUser is the role that is needed.

Reply:
I was thinking that was not ready yet. I've updated the roles